### DISCUSSION TITLE
merge collections into one or init_from more

### DISCUSSION THREAD

**Discussion by sevenjay** [qdrant/qdrant#2799] [Q&A] [answered]
how to merge collections into one or 
cloud init_from specify more than one collection?
like "init_from": ["c1", "c2", "c3"]
v1.6.0 Qdrant

**Comment by timvisee** [comment, marked as answer]
Qdrant currently has no feature to achieve this.

You can do this by setting up a small script though. You can [`scroll`](https://qdrant.tech/documentation/concepts/points/#scroll-points) over all the points in the collections you'd like to merge, and insert them into a single new collection.
